src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,73 @@
+package duke.tasks;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * List that contains all the task.
+ */
+public class TaskList {
+    private List<Task> tasks;
+
+    /**
+     * Constructor to create TaskList.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Overloaded Constructor to create TaskList object when a list is already saved in local storage.
+     *
+     * @param tasks list saved in user's local storage.
+     */
+    public TaskList(List<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Adds a new task to TaskList.
+     *
+     * @param task task to be added into list of tasks.
+     */
+    public void add(Task task) {
+        tasks.add(task);
+    }
+
+    /**
+     * Removes task from TaskList.
+     *
+     * @param index position of task being deleted in the TaskList.
+     */
+    public void delete(int index) {
+        tasks.remove(index);
+    }
+
+    /**
+     * Get task from TaskList.
+     *
+     * @param index position of task needed in the TaskList.
+     * @return task at the index position in TaskList.
+     */
+    public Task get(int index) {
```

Comment:
Is this a good way to name your method? It may be clearer to specify getTaskAtIndex instead. This is so that it would not be confused with find